.travis.yml

```diff
@@ -1,13 +1,38 @@
 dist: trusty
 sudo: required
 language: cpp
+services:
+  - docker
 addons:
+
+env:
+    global:
+        - PROTOBUF_VERSION: 3.6.1
+        - MKLDNN_VERSION: 0.16
+        - MAKE_JOBS: 2
+        # The next declaration is the encrypted COVERITY_SCAN_TOKEN, created
+        #   via the "travis encrypt" command using the project repo's public key
+        - secure: "q1I4YsB5VcNaF9Unmm6T92ht9/KwOGbxZVCpXIp5XUVulpaZq7sTd3rL1v3r1mUCYaabkcy9N4UPQjJZsuOlU4jc8zPzPxPir7hOER5umlkfSMuc1RhmShT8cK9naznqv7FLSTIjTZIao85Lrgxgw0B6xzcWc0kSeJPJVAmS5kwmC/FCQS2MPQpyhfE5JjpUrePOT+lRTB6Psm5bWyEww8bPsatO2k5b8DDdmUJIxmuJ1UTCx5rj/ZcTJLWAsj8D7u9aUfCmOhV5+hqHBvJd/06FLt254SNmvzmVLW9CVU/aZvuTtRECgBYCVndR7NxWpRHo1SBKqgLu+cNOFoFyt++1V+FAbpxj9JMktZNyxWp22c/FvBBdHynOsxBxVFdGIzhcwhQMiHFLOK3pnyiByabtINhERqrszkbpztOepBE3o8PGpjOz8iIx1TtLgmWwAw5D6WXx8FeP5FMkJwpXckCMI5tX5wPoU8cpZIwPjCxG3Z+ojHw+80pQWCrMZnEDfcf9zskJNsmv/GbiWGEvI8xVG0gst5VmjaAXK7JhC0cKvPOEmCFRGY+BWdjD3dkYIIElUmBRfTRDpcDJV6j5r1xMv7QKRFDfAjnC33KLJo2aALZTrkRPveIP2h2jU13ZbemN8GKWwEWNzidmwtCbH4rpe80rFqASWkyfii7HrEI="
+
+cache:
+    directories:
+        - ${HOME}/downloads
+        - ${HOME}/build/protobuf-${PROTOBUF_VERSION}
+        - ${HOME}/build/mkl-dnn-${MKLDNN_VERSION}
+
 matrix:
     include:
         - os: osx
+          env: PLATFORM=macosx-x86_64
         - os: osx
-          env: STATIC=1
+          env: PLATFORM=macosx-x86_64 LINK_STATIC=true
+        - os: linux
+          env: PLATFORM=linux-x86_64 BUILDENV_IMAGE=okapies/buildenv:linux-x64-devtoolset-6
         - os: linux
+          env: PLATFORM=linux-x86_64 BUILDENV_IMAGE=okapies/buildenv:linux-x64-devtoolset-6 LINK_STATIC=true
+        # Use Travis directly instead of container to run static code analysis using Coverity
+        - os: linux
+          env: RUN_COVERITY=true
```

Comment:
If this build matrix entry is just for running Coverity Scan, you might want to skip script section.
How do you think?

Reply:
I fixed to skip the build step and avoid running this job in branches other than `coverity_scan`. It seems to work as expected in the [target branch](https://travis-ci.org/pfnet-research/menoh/jobs/429948062).